clients/client-coinbase/src/index.ts

```diff
@@ -293,13 +294,26 @@ Generate only the tweet text, no commentary or markdown.`;
 		const buy = event.event.toUpperCase() === "BUY";
 		const amountInCurrency = buy
 			? amount * 1e6
-			: (amount / Number(event.price)) * 1e18;
+			: (amount / Number(event.price)) * (event.ticker === "BTC" ? 1e8 : 1e18);
```

Comment:
for now it's okay but going forward if we add more tokens to trade, we should store token precision in another object